Directory.Build.targets

```diff
@@ -8,4 +8,20 @@
     <!-- Avoid compile error about missing namespace when combining ImplicitUsings with .NET Framework target frameworks. -->
     <Using Remove="System.Net.Http" Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'" />
   </ItemGroup>
+
+    <!-- Enable code generation for resx files. -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.ResxSourceGenerator" PrivateAssets="all" />
```

Comment:
Should this be an `Update` rather than an `Include`?

Reply:
No. This is the only Include of this item in the repo. If I changed it to Update, no item would ever be created.